index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Ghibli Review App</title>
+    <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
+    <script src="./index.js"></script>
+    <link rel="stylesheet" href="./master.css" />
+</head>
+
+<body>
+    <section>
+
+        <img
+            src="https://upload.wikimedia.org/wikipedia/en/thumb/c/ca/Studio_Ghibli_logo.svg/1200px-Studio_Ghibli_logo.svg.png" />
+        <h1>
+            Ghibli Review App
+        </h1>
+    </section>
+    <form>
+        
+        <select>
+            <option value="0" selected disabled>Choose Movie</option>
+            <input type="text" value="" placeholder="Enter Your Review" id="textInput" />
+            <div id="movieInfo"></div>
```

Comment:
It would be a little more user friendly if the movie info came right after the select bar, and the text input form came after the movie description.